dubbo-rpc/dubbo-rpc-api/src/main/java/org/apache/dubbo/rpc/SimpleAsyncRpcResult.java

```diff
@@ -44,7 +44,6 @@ public SimpleAsyncRpcResult(CompletableFuture<Object> future, CompletableFuture<
 
     @Override
     public Object recreate() throws Throwable {
-        // TODO should we check the status of valueFuture here?
-        return null;
-    }
+        return valueFuture.get();
+     }
```

Comment:
I think this will block the invocation.   But recreate is used for async invocation.